serverless.yml

```diff
@@ -19,23 +34,20 @@ plugins:
 
 custom:
   client:
-    bucketName: my-store-app
+    bucketName: macrebel-gaming
     distributionFolder: dist
-  s3BucketName: ${self:custom.client.bucketName}
+    indexDocument: index.html
+  s3LocalPath: /
+  s3BucketName: macrebel-gaming
 
-  ## Serverless-single-page-app-plugin configuration:
-  s3LocalPath: ${self:custom.client.distributionFolder}/
-  serverless-s3-cleaner:
-    buckets:
-      - ${self:custom.client.bucketName}
 
 resources:
   Resources:
     ## Specifying the S3 Bucket
     WebAppS3Bucket:
       Type: AWS::S3::Bucket
       Properties:
-        BucketName: ${self:custom.s3BucketName}
```

Comment:
I guess you should use reference ${self:custom.s3BucketName} to your bucket name for avoiding duplicated